LogosLinuxInstaller.py

```diff
@@ -201,6 +205,17 @@ def parse_args(args, parser):
         config.ACTION = control.backup
     elif args.restore:
         config.ACTION = control.restore
+    elif args.update_latest_appimage:
+        if config.WINEBIN_CODE == "AppImage" or config.WINEBIN_CODE == "Recommended":
+            status = utils.compare_recommended_appimage_version()
```

Comment:
`utils.compare_recommended_appimage_version()` returns a tuple, so maybe it should be `status, _ = ...`, if you don't care about the 2nd item and just want to toss it aside.